frontend/src/app/features/invite-user-modal/summary/summary.component.html

```diff
@@ -8,17 +8,19 @@
   <div class="spot-divider"></div>
 
   <div class="spot-modal--body spot-container">
+
     <spot-form-field [label]="text.projectLabel">
-      <p slot="input">{{ project.name }}</p>
+      <p slot="input" class="op-ium-summary-details">{{ project.name }}</p>
```

Comment:
Can you also give all of these an explicit `spot-body-big` (or is it small?) class?

Reply:
Yes sure, the design in Figma is Body Small/Regular